qp_klp/Amplicon.py

```diff
@@ -41,7 +41,9 @@ def quality_control(self):
             output_folder = join(self.pipeline.output_path,
                                  'QCJob',
                                  project_name,
-                                 Step.AMPLICON_TYPE)
+                                 # for legacy purposes, do not replace this
```

Comment:
Can you add more info about this? With the limited text is not clear why this is important.